build/media-suite_compile.sh

```diff
@@ -406,9 +431,8 @@ if [[ $mediainfo = y || $bmx = y || $curl != n ]] &&
     hide_conflicting_libs
     CPPFLAGS+=" -DNGHTTP2_STATICLIB" \
         do_separate_confmakeinstall global "${extra_opts[@]}" \
-        --without-{libssh2,random,ca-bundle,ca-path,librtmp,libidn2} \
-        --with-brotli \
-        --enable-sspi --disable-{debug,manual}
+        --without-{random,ca-bundle,ca-path,librtmp} \
```

Comment:
You should keep libssh2 disabled. It's a troublesome lib to include in static builds.